querqy-core/src/main/java/querqy/rewrite/QueryRewriter.java

```diff
@@ -21,6 +22,19 @@ public interface QueryRewriter {
     
     Set<Term> EMPTY_GENERABLE_TERMS = Collections.emptySet();
 
-    ExpandedQuery rewrite(ExpandedQuery query);
+    RewrittenQuery rewrite(final ExpandedQuery query);
```

Comment:
Let's remove this method altogether right now. We're introducing a breaking change on this method anyway.

Reply:
ok